From f84c197e92c286ecb496045be9b4e97198e59368 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Mon, 21 Jan 2019 14:44:28 +0200
Subject: [PATCH 629/706] enetc: Fix cast to restricted __le64 warning

Should fix:
drivers/net/ethernet/freescale/enetc/enetc_hw.h:310:16: warning: cast
to restricted __le64

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 6545c48fab242ba90b160d4fbb3b687ee44c8e0a)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_hw.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_hw.h b/drivers/net/ethernet/freescale/enetc/enetc_hw.h
index 56ac055cce9f..8cb37f777c20 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_hw.h
+++ b/drivers/net/ethernet/freescale/enetc/enetc_hw.h
@@ -314,7 +314,7 @@ static inline u64 enetc_rd_reg64(void __iomem *reg)
 		tmp = ioread32(reg + 4);
 	} while (high != tmp);
 
-	return le64_to_cpu((u64)high << 32 | low);
+	return le64_to_cpu((__le64)high << 32 | low);
 }
 #endif
 
-- 
2.17.1

